src/app/_utils/code-example.ts

```diff
@@ -5,7 +5,7 @@
  */
 
 import {Component, Input, ViewChild} from "@angular/core";
-import {CodeHighlight} from "../../clarity-angular/code/code-highlight";
+import {SyntaxHighlight} from "../../clarity-angular/code/syntax-highlight/syntax-highlight";
```

Comment:
@adityarb88 thanks, the file name was supposed to change. The class name should stay the same.